bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml

```diff
@@ -8,12 +8,9 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" name="org.eclipse.smarthome.ui.chart">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.smarthome.ui.chart">
```

Comment:
I used this one because you have done it the same way in the icon servlet.
I suspected a...=a..., d...=d..., m...=m... is the default one.